### PR TITLE
Make a copy of the nn.Module object for conversion

### PR DESCRIPTION
**Pull request by sharath**
The conversion toolkit shouldn't modify the nn.Module object. There might be a better way to accomplish the same thing though.

I also removed the print statements for the time elapsed.

### CODE REVIEW

**Comment by sharath** [comment]
The conversion turns off autograd in data_based_normalization()
[Here's a demo of what I mean](https://gist.github.com/sharath/3be4eed95ead96cdd1e0d385659a8f10)

**Comment by dee0512** [comment]
@sharath ok. A simple fix is ann = torch.tensor(ann). No need to deepcopy. And why remove the logs?

**Comment by sharath** [comment]
It doesn't seem necessary? I think it makes more sense to issue a RuntimeWarning when data is None actually

**Comment by dee0512** [comment]
I agree that the time elapsed isn't necessary but the "starting data normalization" is helpful since it helps debug when there is an error. I have had my programs stuck after that data normalization multiple times. Including a warning is good too. It would help inform that the weights are not being scaled.
